manifests/0000_90_kube-controller-manager-operator_04_servicemonitor-controller-manager.yaml

```diff
@@ -78,3 +78,46 @@ spec:
   selector:
     matchLabels:
       prometheus: kube-controller-manager
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: kube-controller-manager
+  name: cluster-policy-controller
+  namespace: openshift-kube-controller-manager
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  endpoints:
+    - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      interval: 30s
+      metricRelabelings:
+        - action: drop
+          regex: etcd_(debugging|disk|request|server).*
+          sourceLabels:
+            - __name__
+        - action: drop
+          regex: rest_client_request_latency_seconds_(bucket|count|sum)
```

Comment:
I don't know when this has changed , but is `rest_client_request_duration_seconds` IIUIC https://github.com/kubernetes/kubernetes/blob/17381762dd6caafe72cda9e9c39e42405391f1e9/staging/src/k8s.io/component-base/metrics/prometheus/restclient/metrics.go#L36

Reply:
thanks for noticing, yes it seems to have been removed 2 years ago https://github.com/kubernetes/kubernetes/commit/606e4503cf4cea9f05cfd467b88c88cf9c5648d9

I have added `rest_client_request_duration_seconds` to be dropped